std_msgs/Header header
uint8 src_sysid
uint32 seq
uint8 tid
uint8[] payload